win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1279,11 +1223,34 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont)
             /* Set face */
             FontGDI->SharedFace = SharedFace;
             FontGDI->CharSet = ANSI_CHARSET;
-            FontGDI->OriginalItalic = ItalicFromStyle(Face->style_name);
+            FontGDI->OriginalItalic = FALSE;
             FontGDI->RequestItalic = FALSE;
-            FontGDI->OriginalWeight = WeightFromStyle(Face->style_name);
+            FontGDI->OriginalWeight = FW_NORMAL;
             FontGDI->RequestWeight = FW_NORMAL;
 
+            if (FT_IS_SFNT(Face))
+            {
+                IntLockFreeType();
```

Comment:
I would move the locking to outside the `if (FT_IS_SFNT(Face))` block (so just lock this entire new block).
As for why Italic does not work, not sure :(

Reply:
Maybe `FT_GlyphSlot_Oblique` doesn't support bitmap glyphs. Should we manually transform the bitmap glyphs?

Reply:
We can use `FT_LOAD_NO_BITMAP` flag to load non-bitmap glyphs.

Reply:
Gotcha!
![gotcha](https://user-images.githubusercontent.com/2107452/69234070-883d2380-0bd1-11ea-83ee-3a2790279909.png)